finance-info.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index d51267a..994b8c5 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -3,19 +3,18 @@ name = Finance
+ activity_version = 14
```

Comment:
Let's start using `14.1`, to show we are a bit ahead since the last official release. (I need to update this myself in my original ports).